Cargo.toml

```diff
@@ -24,26 +24,10 @@ glob = "0.3.0"
 cairo-felt = "0.8.2"
 
 # Cairo runner dependencies
-cairo-lang-runner = {git = "https://github.com/starkware-libs/cairo", tag = "v2.2.0"}
-cairo-lang-test-runner = {git = "https://github.com/starkware-libs/cairo", tag = "v2.2.0"}
-cairo-lang-compiler = {git = "https://github.com/starkware-libs/cairo", tag = "v2.2.0"}
-cairo-lang-casm = {git = "https://github.com/starkware-libs/cairo", tag = "v2.2.0"}
-cairo-lang-diagnostics = {git = "https://github.com/starkware-libs/cairo", tag = "v2.2.0"}
-cairo-lang-debug = {git = "https://github.com/starkware-libs/cairo", tag = "v2.2.0"}
-cairo-lang-defs = {git = "https://github.com/starkware-libs/cairo", tag = "v2.2.0"}
-cairo-lang-sierra = {git = "https://github.com/starkware-libs/cairo", tag = "v2.2.0"}
-cairo-lang-sierra-ap-change = {git = "https://github.com/starkware-libs/cairo", tag = "v2.2.0"}
-cairo-lang-sierra-gas = {git = "https://github.com/starkware-libs/cairo", tag = "v2.2.0"}
-cairo-lang-sierra-generator = {git = "https://github.com/starkware-libs/cairo", tag = "v2.2.0"}
-cairo-lang-semantic = {git = "https://github.com/starkware-libs/cairo", tag = "v2.2.0"}
-cairo-lang-sierra-to-casm = {git = "https://github.com/starkware-libs/cairo", tag = "v2.2.0"}
-cairo-lang-utils = {git = "https://github.com/starkware-libs/cairo", tag = "v2.2.0"}
-cairo-lang-filesystem = {git = "https://github.com/starkware-libs/cairo", tag = "v2.2.0"}
-cairo-lang-starknet = {git = "https://github.com/starkware-libs/cairo", tag = "v2.2.0"}
-cairo-lang-syntax = {git = "https://github.com/starkware-libs/cairo", tag = "v2.2.0"}
-cairo-lang-plugins = {git = "https://github.com/starkware-libs/cairo", tag = "v2.2.0"}
-cairo-lang-lowering = {git = "https://github.com/starkware-libs/cairo", tag = "v2.2.0"}
-
+cairo-lang-test-runner = {git = "https://github.com/starkware-libs/cairo", tag = "v2.3.0"}
+cairo-lang-test-plugin = {git = "https://github.com/starkware-libs/cairo", tag = "v2.3.0"}
+scarb = { git = "https://github.com/software-mansion/scarb", version = "2.3.0" }
```

Comment:
why a direct dependency rather then just requiring Scarb installed in the setup instructions? Now that it can be done with a one-line in the terminal perhaps that'd be better

Reply:
This is for new people on boarding to Starknet/Cairo.
Be lesser the setup required the better IMO.

Reply:
I feel it's nice to have minimal setup for devs looking forward to playing with Cairo.